test/C/c/sendreceive.c

```diff
@@ -4,11 +4,11 @@
 #include "../include/IntPrint/Check.h"
 #include "../include/api/set.h"
 
-void send(print_self_t* self, print_out_t* out) {
+void sender(print_self_t* self, print_out_t* out) {
```

Comment:
Yes, this is because name collisions. I think that the next step here is to fix this problem by re-architecting the platform support files such that we can include "platform.h" without including e.g. "windows.h" and "pthread.h". 

I think this re-architecting should be a separate PR.